src/EditorFeatures/VisualBasic/QuickInfo/SemanticQuickInfoProvider.vb

```diff
@@ -92,6 +92,13 @@ Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.QuickInfo
             Return Await MyBase.BuildContentAsync(document, token, cancellationToken).ConfigureAwait(False)
         End Function
 
+        ''' <summary>
+        ''' Given a Sub or Function token for a lambda, returns the syntax for the whole lambda
+        ''' </summary>
```

Comment:
do we have tests for single/multiline sub/function lambdas?

Reply:
Yes, I have all four cases